tensornetwork/backends/numpy/numpy_backend_test.py

```diff
@@ -0,0 +1,113 @@
+"""Tests for graphmode_tensornetwork."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+import tensornetwork
+import pytest
+
+
+def test_tensordot():
+  net = tensornetwork.TensorNetwork('numpy')
```

Comment:
Instead of making a `TensorNetwork`, you can just construct the backend object directly and use that. That is preferred since it decouples the backend tests from the implementation of `TensorNetwork`.

You can do this by doing

```
from tensornetwork.backends.numpy import numpy_backend
backend = numpy_backend.NumPyBackend()
```

Reply:
Totally right